spec.html

```diff
@@ -36422,10 +36997,14 @@ <h1>
           <p>This section is amended in <emu-xref href="#sec-parsepattern-annexb"></emu-xref>.</p>
         </emu-note>
         <emu-alg>
-          1. If _u_ is *true*, then
-            1. Let _parseResult_ be ParseText(_patternText_, |Pattern[+UnicodeMode, +N]|).
+          1. If _v_ is *true* and _u_ is *true*, then
+            1. Let _parseResult_ be a List containing one *SyntaxError* object.
```

Comment:
Why must it be only one?

```suggestion
            1. Let _parseResult_ be a List containing one or more *SyntaxError* objects.
```

Reply:
PTAL: https://github.com/mathiasbynens/ecma262/pull/21